### ISSUE TITLE
Is Cornelius afraid of 0's?

### ISSUE THREAD

**Issue by ChaelCodes** [ChaelCodes/CorneliusCodes#7]
# Description
I've been watching Cornelius for a while, and I think Cornelius is afraid of the 0 location on the x and y axis of the board!

![cornelius wiggles away from zero](https://user-images.githubusercontent.com/8124558/134375357-04b0ef2c-183b-4a1e-835b-3370a6d06d8b.gif) ![cornelius would rather eat themself than go onto the 0 x axis.](https://user-images.githubusercontent.com/8124558/134375817-835d7f5d-f627-41f3-9690-5510104709a1.gif) ![more wiggly cornelius avoiding 0 on x and y](https://user-images.githubusercontent.com/8124558/134376329-16b2f23f-39ae-4093-8ea7-5632f8467f29.gif) ![Cornelius avoids scary 0s and runs into the wall](https://user-images.githubusercontent.com/8124558/134377640-ee946322-48e4-44ba-8a3e-15419ec8b9e7.gif) ![more wiggly cornelius](https://user-images.githubusercontent.com/8124558/134378062-34f9d7a8-73fd-4bdd-b26a-7e977694c41f.gif)

This makes perfect sense. Cornelius is a very superstitious snek, and 0 is an unlucky number. Dividing by zero is death for a Battlesnake, and everyone knows most off-by-one errors happen near 0.

# Changes
What can we do to help Cornelius feel safe on the 0 x/y axis?

**Comment by PinkFlufflyLlama** [comment]
https://github.com/ChaelCodes/CorneliusCodes/blob/e0c4fad2365519619cff15ad6382416dd06b865c/src/logic.rs#L97 does this not need to be -1 because what that is saying is that 0 is an illigal move

**Comment by ChaelCodes** [comment]
Yeah!! That's what I did wrong. 🤣  @matthewd actually mentioned it several times on stream, but I didn't write a todo and procrastinated on fixing it. At the end of the stream, I totally forgot. So now CorneliusCodes is canonically afraid of 0.

**Comment by PinkFlufflyLlama** [comment]
the poor love

**Comment by PinkFlufflyLlama** [comment]
What I might do given I have some time, I'll write a contingency plan for corny to follow if he gets stuck forced out of his comfort zone

**Comment by PinkFlufflyLlama** [comment]
My solution is going to allow the creation of various "contingency plans" that get used if safe route is found, and it will run through the plans, trying to plan to see if there is a possible escape at all. I'm going to implement just the 0, 0 plan, but some other planes that could be added for Corny to follow are: Move to where a snake tail end is, and Move to where a snake head could be ( since potential head spots are going to be made illegal )

I see this as a way to add some heuristic scrambling where behavior defined here may not succeed in all cases, but is generally better than not submitting a move (as currently if the .unwrap() on the move fails, it throws it)
